lang/po/es_ES.po

```diff
@@ -156204,7 +156204,7 @@ msgid ""
 msgstr ""
 "Al ignorar la ofensiva a favor de la autodefensa, eres mejor en la protección.\n"
 "\n"
-"Daño bloqueado reducido en un 100%% de Destreza."
```

Comment:
Unless I'm missing something, this %% was necessary in order to print a % to the final string.

Reply:
I guess this is a plain unformatted string, because in the original English string there is only a single %